src/main/java/net/snowflake/ingest/streaming/internal/ChannelCache.java

```diff
@@ -43,6 +56,46 @@ void addChannel(SnowflakeStreamingIngestChannelInternal<T> channel) {
     }
   }
 
+  /**
+   * Get the last flush time for a table
+   *
+   * @param fullyQualifiedTableName fully qualified table name
+   * @return last flush time in milliseconds
+   */
+  Long getLastFlushTime(String fullyQualifiedTableName) {
```

Comment:
Optional<Long> to force the caller to make sure that what's returned is not null?

Reply:
Changed to `Optional<Long>` and verified at caller site.